README.md

```diff
@@ -157,13 +157,13 @@ release.
 
 Additionally, as part of each release detailed release notes are written to
 document in detail what has changed as part of a release. This includes any
-documentation on potential breaking changes on upgrade and new features.
+documentation on potential breaking changes on upgrades and new features.
 For example, you can find the release notes for the `0.9.0` release in the
 Qiskit documentation here:
 
 https://qiskit.org/documentation/release_notes.html#terra-0-9
 
-## Acknowledgements
+## Acknowledgments
```

Comment:
Both of these are valid spellings of the word, I'm not sure we need to change it.